reference/orders.v2.oas2.yml

```diff
@@ -485,6 +485,25 @@ paths:
                     country: United States
                     country_iso2: US
                     email: janedoe@example.com
+              Gift Certificate Consignment:
+                value:
+                  customer_id: 11
+                  consignments:
+                    email:
+                      gift_certificates:
+                        - recipient_email: jane@example.com
+                          line_items:
+                            gift_certificate_id: 184
```

Comment:
I must've missed this before. This is a different format to our GET isn't it? IIRC, we had the URL to the order product and then to get the GC id you needed to seperately query the order product to get the GC id column

Reply:
I think the current format for `GET` is correct as the `POST` creates an `order_product` which references the gc, in the same way creating an order with a catalog product id will create an `order_product` which references the product